src/lib.rs

```diff
@@ -305,6 +305,15 @@ pub trait Rng {
     /// Return the next random f32 selected from the half-open
     /// interval `[0, 1)`.
     ///
+    /// This uses a technique described by Saito and Matsumoto at
```

Comment:
Can you drop a reference to the paper here? Is it [this](http://www.math.sci.hiroshima-u.ac.jp/~m-mat/MT/ARTICLES/dSFMT.pdf)?